pkg/cmd/miscellaneous.go

```diff
@@ -274,18 +276,18 @@ func getEmail(githubURL string) string {
 	if githubURL == "" {
 		return "null"
 	}
-	res, err := ExecCmdReturnOutput("bash", "-c", "curl -ks "+githubURL+"/api/v3/users/"+os.Getenv("USER"))
+	baseURL, err := url.Parse(githubURL)
+	checkError(err)
+	baseURL.Path += "/api/v3/users/"
+	baseURL.Path += os.Getenv("USER")
```

Comment:
```suggestion
	baseURL.Path += url.PathEscape(os.Getenv("USER"))
```
path escape was missing

Reply:
thanks！ @petersutter